examples/servicebus-migration-test/step2/index.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+
+const exampleResourceGroup = new azure.core.ResourceGroup("example", {
+    location: "West Europe",
+    name: "my-servicebus",
+});
+const exampleNamespace = new azure.servicebus.Namespace("example", {
+    location: exampleResourceGroup.location,
```

Comment:
The preference is to avoid assigning locations explicitly and do so from the config.